lib/application.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Application
+ * @constructor
+ * @param {Object} client Catapult client
+ */
+
+var Application = function (client) {
+	/**
+	 * List the user's applications
+	 * @param {Object} params Parameters for filtering applications
+	 * @param {Number} [params.page] The specified page requested when querying
+	 * a list of applications
+	 * @param {Number} [params.size] The size of each page requested
+	 * @param {Function} [callback] A callback for the list of applications
+	 * @return {MessageResponse} A promise for the list of applications
+	 */
+	this.list = function (params, callback) {
```

Comment:
Why is the list function an instance of Application? I shouldn't have to create an application to list my applications